app/controllers/course/admin/assessment_settings_controller.rb

```diff
@@ -17,7 +17,7 @@ def update
   private
 
   def category_params
-    params.require(:course).permit(assessment_categories_attributes:
+    params.require(:course).permit(:enable_public_test_cases_output, assessment_categories_attributes:
```

Comment:
call this `show`_public_test_cases_output` too?

Reply:
yeah I am modifying the name, and putting it under the settings tree now.